FirebaseSessions/Tests/Unit/Mocks/ShadowDate.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ShadowDate.swift
+//  Pods
+//
+//  Created by Leo Zhan on 2022-10-17.
+//
+
+import Foundation
+
+// A paused Date that can only be advanced through functions
+class ShadowDate {
```

Comment:
I'm curious why Shadow- instead of Mock-? As far as I know about the terms, Shadow inspects the inside of the class vs mock just mocks functionality.

Reply:
Done, changed to Mock-